src/curated-corpus/helpers/definitions.ts

```diff
@@ -67,3 +67,31 @@ export const curationStatusOptions: DropdownOption[] = [
 export type ApprovedItemFromProspect = Omit<ApprovedCorpusItem, 'language'> & {
   language: CorpusLanguage | undefined;
 };
+
+// scheduled surface guid to timezone map
+export const guidToUtcOffset = [
```

Comment:
do we need this map now that the shared data construct has been updated to include `ianaTimezone`?